### DISCUSSION TITLE
How can I recover  Sector?

### DISCUSSION THREAD

**Discussion by hsjw12** [filecoin-project/lotus#6058] [Storage Provider] [answered]
> Note: For security-related bugs/issues, please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
![20210417_011342](https://user-images.githubusercontent.com/70571701/115053529-30b3fc00-9f1a-11eb-9c36-d7f0e22173d5.png)

Can I recover sectors that have become like the contents of the image?
Please help me...

>version
Daemon:  1.7.0-dev+mainnet+git.8a19ddbb9+api1.1.0
Local: lotus version 1.7.0-dev+mainnet+git.8a19ddbb9

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
are you sure these  sectors didn't expire?

**Reply by hsjw12** [reply]
The file in storoe was deleted and could not be restored.
